app/src/androidTest/java/com/github/sdp/mediato/DatabaseTests/CollectionsTests.java

```diff
@@ -69,18 +71,18 @@ public void setUp() throws ExecutionException, InterruptedException, TimeoutExce
 
     @AfterClass
     public static void cleanDatabase() {
-      Database.database.getReference().setValue(null);
+      UserDatabase.database.getReference().setValue(null);
     }
 
     @Test
     //Tests that the collections are added, retrieved and removed properly
     public void addsRetrievesAndRemovesCollectionProperly() throws ExecutionException, InterruptedException, TimeoutException {
         //Adds the collection
-        Database.addCollection(user1.getUsername(), collection1);
+        CollectionsDatabase.addCollection(user1.getUsername(), collection1);
         Thread.sleep(1000);
 
         //Test retrieving the collection
-        Collection retrievedCollection = Database.getCollection(user1.getUsername(), collection1.getCollectionName())
+        Collection retrievedCollection = CollectionsDatabase.getCollection(user1.getUsername(), collection1.getCollectionName())
                 .get(STANDARD_COLLECTION_TIMEOUT, TimeUnit.SECONDS);
         assertEquals(collection1.getCollectionName(), retrievedCollection.getCollectionName());
         assertEquals(collection1.getCollectionType(), retrievedCollection.getCollectionType());

```

Comment:
Is it possible to split this test into two different ones (one for addition and one for deletion) ?

Reply:
Yes good catch :)
Addressed in commit 82c8183